source/base/TimeSpan.ooc

```diff
@@ -54,14 +54,14 @@ TimeSpan: cover {
 	//  %h - hours (<24)
 	//  %m - minutes (<60)
 	//  %s - seconds (<60)
-	//  %z - milliseconds (<1000)	
+	//  %z - milliseconds (<1000)
```

Comment:
This is a copy of the line below for some reason, so just remove the whole thing.